docs/index.md

```diff
@@ -118,6 +118,8 @@ render((
 ), document.getElementById("app"));
 ```
 
+> Note: If there are fields in the `formData` that are not represented in the schema, they will be retained by default. If you would like to omit those unused values on form submission, set the `omitUnusedData` prop to `true`.
+
```

Comment:
@travisdahl can you document the `name` prop for all the widgets/fields somewhere in the docs?